test/jest/__snapshots__/FullSave.test.ts.snap

```diff
@@ -548,6 +548,29 @@ exports[`Check Save File Continuity FactionsSave continuity 1`] = `
 }
 `;
 
+exports[`Check Save File Continuity GoSave continuity 1`] = `
```

Comment:
Optional: Maybe have something other than a *completely* fresh game to test more of the save/load surface?